transport/http/src/test/scripts/org/kaazing/gateway/transport/http/test.http.handshake.timeout.rpt

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect tcp://localhost:8001
+connected
+
+read await BARRIER
+write "GET /clientaccesspolicy.xml HTTP/1.1\r\n"
+write "Accept-Language: en-US\r\n"
+write "User-Agent: Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; WOW64; Trident/6.0)\r\n"
+write "Host: localhost:8001\r\n"
+write "\r\n"
+
+read "HTTP/1.1 404 Not Found\r\n"
+
+close
```

Comment:
Based on the log I think the sever may be closing the connection here, which is sort of expected.  I'm not even sure if a 404 is the correct error code for a slow handshake timeout.  See http://www.restapitutorial.com/httpstatuscodes.html. A 408 seems more appropriate